packages/OntologyService/README.md

```diff
@@ -3,7 +3,7 @@
 Simple client-side library for working with RDF ontologies. Includes basic CRUD
 abilities and functions to help you navigate an ontology hierarchy.
 
-For further information on how to use the library see the [API Documentation](https://somehting.com)
+For further information on how to use the library see the [API Documentation](README.md#API)
 
 ## Install
```

Comment:
As per the main README.md, in the Usage section, below, the code example just needs a one-liner to introduce it, e.g. "A simple example to get all diagrams from the ontology service:".